.github/cctesttool-publish.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish package to GitHub Packages
+on:
+  release:
```

Comment:
I'm not a GitHub Actions expert, but on release doesn't sound right to me. We don't cut releases of fabric-test. And I don't think we would start cutting releases of the global project for the purpose of publishing one tool in the repository. Maybe I'm misunderstanding the intent though.

Perhaps we should publish as part of the nightly interop job, similar to the publishing of the other assets from this repo. Or perhaps publish upon merge if there happens to be a change within the /tools/chaincode-integration directory.

Reply:
Ah .. I used "release" just because things in other repos have a publish cycle; there was no intent to do any other than make the code available 'based on some suitable action'

 Publish on merge makes sense.  I can update the PR to be merge.